packages/github/src/interfaces.ts

```diff
@@ -1,39 +1,29 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
+// TODO: https://github.com/actions/toolkit/issues/291, ESLint chokes on the | syntax
+/* eslint-disable no-undef */
+import Webhooks from '@octokit/webhooks'
+type WebhookPayload =
+  | Webhooks.WebhookPayloadPush
```

Comment:
The event corresponds to the name pretty cleanly here so I chose not to add comments 
WebhookPayloadPush = "push" event